### PR TITLE
Change the primary UI font, darken the background, and show template icons for workspaces

### PR DESCRIPTION
**Pull request by kylecarbs**
The title mentions it all! Check out storybook to see the changes.

image

#### Why change the font family?

Our UI looked very skunkworks. By adjusting the primary font to Inter, it becomes a lot more readable. This also aligns with more Material UI defaults, which is healthy for the codebase!

#### Why darken the background?

The dashboard was a light-dark mode, which put it in a weird spot. It wasn't very dark but wasn't very light. This adjusts the shades by 2, which makes the whole UI look a bit sharper from my perspective.

#### Why add icons on the workspaces page?

This change could be done in a separate PR, but I think it sharpens the UI visually. It's odd to have a primary UI indicator inset on the second column.

### CODE REVIEW

**Comment by BrunoQuaresma** [comment]
I liked the changes. I just think the background is too dark if compared with the other component. What do you think?

I also would wait for this PR be merged from @presleyp https://github.com/coder/coder/pull/3842

**Comment by presleyp** [comment]
My PR is in now! I think a darker background will make it easier to get good color contrast. But I think some of the color contrast might be a little *too* strong now - what do you think about changing the `colors.gray[3]`s out for 4s in the palette?

**Comment by BrunoQuaresma** [comment]
@presleyp I think there are too many changes on Chromatic to go one by one, are you ok if we just accept all this time?

**Review by BrunoQuaresma — APPROVED** [review]
If you think that is ok, I'm good with that. Maybe would be worth seeing what it looks like with the @presleyp suggestion.

**Comment by presleyp** [comment]
@BrunoQuaresma yeah I looked at chromatic and found it most helpful to look at the pages, not the individual components. But we want to make sure that the next PR doesn't have to review changes left over from this one. Is there a shortcut we can take?

File: site/src/components/Workspace/Workspace.tsx [hunk 1]
```diff
@@ -104,8 +106,15 @@ export const Workspace: FC<React.PropsWithChildren<WorkspaceProps>> = ({
         }
       >
         <WorkspaceStatusBadge build={workspace.latest_build} className={styles.statusBadge} />
-        <PageHeaderTitle>{workspace.name}</PageHeaderTitle>
-        <PageHeaderSubtitle>{workspace.owner_name}</PageHeaderSubtitle>
+        <Box display="flex">
+          {hasTemplateIcon && (
+            <img alt="" src={workspace.template_icon} className={styles.templateIcon} />
```
Comment (jsjoeio) [inline site/src/components/Workspace/Workspace.tsx:111]: I think for now it's okay to use a generic alt string
```suggestion
            <img alt="Template icon" src={workspace.template_icon} className={styles.templateIcon} />
```

**Review by jsjoeio — APPROVED** [review]

**Comment by BrunoQuaresma** [comment]
@presleyp yes. You can accept the build instead of a single component.

Screen Shot 2022-09-06 at 13 37 24

**Comment by kylecarbs** [comment]
@presleyp I swapped em!

**Pull request merged**